src/app/exercise/tests/tests.component.css

```diff
@@ -1,35 +1,35 @@
 .test:before {
-  width: 10px;
-  height: 10px;
-  border-radius: 50%;
-  display: inline-block;
-  background-color: #dddddd;
-  content: "";
-  margin-right: 8px;
+    width: 10px;
```

Comment:
Mind trying https://github.com/editorconfig/editorconfig-vscode plugin to see if it'll make the formatting more consistent?